app/views/layouts/landing_page_layout.html.erb

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>
-    Help for early years providers
+    Help for early years providers - GOV.UK
   </title>
```

Comment:
This is why it is not a good idea to repeat phrases in the page title

![Screenshot 2021-05-18 at 18 24 48](https://user-images.githubusercontent.com/3352035/118696844-9be93a80-b806-11eb-889d-588029ad969e.png)